roles/contiv_network/tasks/ovs.yml

```diff
@@ -52,3 +56,11 @@
   with_items:
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
+
+- name: disable SE linux
```

Comment:
I understand the implications of disabling SE linux globally.
But, when SE linux is enabled, OVS can not connect to netplugin(OVS fails to establish controller connection). If anybody knows what might be causing it, i'm willing to try.